sei/chain.json

```diff
@@ -0,0 +1,118 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "sei",
+    "status": "live",
+    "website": "https://www.sei.io/",
+    "network_type": "mainnet",
+    "pretty_name": "Sei",
+    "chain_id": "pacific-1",
+    "bech32_prefix": "sei",
+    "daemon_name": "seid",
+    "node_home": "$HOME/.sei",
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "usei",
+                "fixed_min_gas_price": 0,
+                "low_gas_price": 0,
+                "average_gas_price": 0.025,
+                "high_gas_price": 0.04
+            }
+        ]
+    },
+    "staking": {
+        "staking_tokens": [
+            {
+                "denom": "usei"
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/sei-protocol/sei-chain",
+        "recommended_version": "v3.0.0",
```

Comment:
the sei release versions dont have `v` prefixing, and the recommended / latest version is `3.0.8`

https://github.com/sei-protocol/sei-chain/releases/tag/3.0.8

Reply:
we may make a change in the future to have `v` prefixing for best practice, but as of now that isn't the format